src/contrib/cluster/Akka.Cluster.Sharding/reference.conf

```diff
@@ -207,6 +207,9 @@ akka.cluster.sharding {
   distributed-data {
     # minCap parameter to MajorityWrite and MajorityRead consistency level.
     majority-min-cap = 5
+    
+    # ShardCoordinator is singleton running on oldest
+    prefer-oldest = on
```

Comment:
`prefer-oldest` should have been enabled for the `DDataShardCoordinator` when https://github.com/akkadotnet/akka.net/pull/5146 was originally introduced, but I overlooked it. This should play a key factor in https://github.com/akkadotnet/akka.net/issues/6973 - where `LWWRegister` data from the `DDataShardCoordinator` is going missing when the coordinator is pushing updates as it's exiting the cluster.